spec/models/message_spec.rb

```diff
@@ -1,5 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe Message, :type => :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it{should validate_presence_of :name}
+  it{should validate_presence_of :email}
+  it{should validate_presence_of :body}
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.